test/extended/builds/pipeline_origin_bld.go

```diff
@@ -166,9 +166,10 @@ var _ = g.Describe("[sig-builds][Feature:Builds][sig-devex][Feature:Jenkins][Slo
 			j = jenkins.NewRef(oc)
 
 			g.By("wait for jenkins to come up")
-			_, err = j.WaitForContent("", 200, 10*time.Minute, "")
+			resp, err := j.WaitForContent("", 200, 5*time.Minute, "")
 
 			if err != nil {
+				g.By(fmt.Sprintf("wait for jenkins to come up got err and resp string %s, dumping pods", resp))
```

Comment:
I feel like this should not be a g.By, and it should probably print out the err and the resp?

Reply:
yeah I waffled on the g.By vs. e2e.Logf ... I'll switch it

Reply:
and add err